### PR TITLE
core(audits): point PWA audit description links to web.dev

### PR DESCRIPTION
**Pull request by mfriesenhahn**
**Summary**
Points "Learn more" links in PWA audits to relevant web.dev guides.

**Note**
web.dev currently doesn't have a post for the **Provides a valid apple-touch-icon** audit. We'll try to draft one quickly.

### CODE REVIEW

File: lighthouse-core/audits/redirects-http.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class RedirectsHTTP extends Audit {
       title: 'Redirects HTTP traffic to HTTPS',
       failureTitle: 'Does not redirect HTTP traffic to HTTPS',
       description: 'If you\'ve already set up HTTPS, make sure that you redirect all HTTP ' +
-         'traffic to HTTPS. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/http-redirects-to-https).',
+         'traffic to HTTPS. [Learn more](https://web.dev/redirects-http).',
```
Comment (patrickhulce) [inline lighthouse-core/audits/redirects-http.js]: this doc seems to give the opposite advice, we *want* them to redirect their http traffic to https and the doc is mostly talking about avoiding redirects

**Comment by patrickhulce** [comment]
Oh and I don't know if we plan to move the PWA checklist over too or not

**Comment by exterkamp** [comment]
I'm wondering @paulirish if `swap-locale-test` should roundtrip from `en-XL` instead of `es`?  Will it fail like this if we change links/change strings every time?

**Comment by brendankenny** [comment]
> I'm wondering @paulirish if `swap-locale-test` should roundtrip from `en-XL` instead of `es`? Will it fail like this if we change links/change strings every time?

shouldn't it be ok and this just needs an `update:sample-json`? Since the swap is by message id, the en and es will just have different links, but they'll still trade back and forth

**Comment by brendankenny** [comment]
yeah, we just checked and @mfriesenhahn you'll just need to run `yarn update:sample-json` to get the strings updated across the needed files

**Comment by mfriesenhahn** [comment]
> Do we want to add one for the PWA overall "Learn more" link?

I discussed with @exterkamp, and we were thinking it'd make sense to leave these as-is for now until we get the relevant info migrated to web.dev. Afterward, we can point those links to the landing page for each collection, if that SGTY.

**Comment by mfriesenhahn** [comment]
@patrickhulce, I've copied your content comments to [web.dev issue 1343](https://github.com/GoogleChrome/web.dev/issues/1343) for tracking.

**Pull request merged**